src/controllers/OrdersController.php

```diff
@@ -99,7 +99,7 @@ protected function getOrders()
         $end_date = $this->parseDate('end_date');
 
         if ($start_date && $end_date) {
-            $query->dateOrdered(array('and', '> '.$start_date, '< '.$end_date));
+            $query->dateUpdated(array('and', '> '.$start_date, '< '.$end_date));
```

Comment:
I'd prefer to keep this as `dateOrdered`. The `dateUpdated` field can is updated for any change to the order and I feel it would be unreliable for a large number of use cases.

Reply:
Thats fine, i'll just use this forked version until your new version is ready. If you could you add in the ability to customise/change the order query from another plugin that would be great.